src/cmds/pbsdsh.c

```diff
@@ -204,6 +306,7 @@ main(int argc, char *argv[], char *envp[])
 #endif
 	extern int optind;
 	extern char *optarg;
+	char *targethost=NULL;
```

Comment:
Our coding standard has spaces around the equals

Reply:
sure, I'll have a check at assignment else where, too.